main.js

```diff
@@ -160,4 +160,244 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// loops using the map method extracting the types into a new array
+const tvTypes = inventory.map((item) => {
+  return item.name
+})
+
+function listSoldOut() {
+  // loops using the filter method looking for soldout items
+  return inventory.filter((item) => {
+    // checks if items sold equals the original stock before returning as soldOut
+    if (item.originalStock - item.sold === 0) {
+      return item
+    }
+  })
```

Comment:
De filter functie verwacht een eigenlijk ten alle tijden een boolean terug te krijgen. Uiteindelijk zal `undefined` vertaald worden naar `false` en alles wat niet `null` of `undefined` is naar `true`, maar het is toch altijd goed om gewoon een boolean terug te geven, omdat dat expliciet is
```suggestion
  return inventory.filter((item) => {
    return item.originalStock === item.sold
  })
```